### PR TITLE
release: v8.6.2

### PR DESCRIPTION
**Pull request by ShGKme**
## [v8.6.2](https://github.com/nextcloud-libraries/nextcloud-vue/tree/v8.6.1) (2024-02-07)
[Full Changelog](https://github.com/nextcloud-libraries/nextcloud-vue/compare/v8.6.1...v8.6.2)
### 📝 Notes
* In `NcRichContenteditable` properties `autoComplete.title`/`userData.title` were renamed back to `label` to be in line with server API. The `title` property is still working as a fallback, but deprecated. See https://github.com/nextcloud-libraries/nextcloud-vue/pull/5215

### 🐛 Fixed bugs
* fix(NcHeaderMenu): button size regression on small viewports fixed by @emoral435 in https://github.com/nextcloud-libraries/nextcloud-vue/pull/5149
* fix(NcListItem): Remove actions from within `a` which caused invalid HTML output by @susnux in https://github.com/nextcloud-libraries/nextcloud-vue/pull/5198
* fix(NcRichContenteditable): make autocomplete accessible by @ShGKme in https://github.com/nextcloud-libraries/nextcloud-vue/pull/4904
* fix(NcRichContenteditable): unify styles and add contrast border on keyboard navigation by @ShGKme in https://github.com/nextcloud-libraries/nextcloud-vue/pull/5214
* fix(NcRichContenteditable): bring back label for autocomplete by @ShGKme in https://github.com/nextcloud-libraries/nextcloud-vue/pull/5215
* fix(NcAppSidebar): move focus to sidebar on open and auto return focus on close by @ShGKme in https://github.com/nextcloud-libraries/nextcloud-vue/pull/5219

### Other Changes
* chore(NcSelect): Mention labelOutside for external label use by @Pytal in https://github.com/nextcloud-libraries/nextcloud-vue/pull/5211

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,23 @@
 
 All notable changes to this project will be documented in this file.
 
-## [v8.6.1](https://github.com/nextcloud-libraries/nextcloud-vue/tree/v8.6.1) (2024-02-01)
```
Comment (Antreesy) [inline CHANGELOG.md:5]: Have you lost a previous release links intentionally?)

**Review by Antreesy — APPROVED** [review]
Nothing blocking from my side, apart from mentioned lines. Will test thoroughly during lib bump

**Review by JuliaKirschenheuter — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ShGKme) [reply CHANGELOG.md:5]: Oooops. Nice catch 💙

Comment (ShGKme) [reply CHANGELOG.md:5]: Fixed 🙈

Comment (Antreesy) [reply CHANGELOG.md:5]: And Full changelog from 8.6.1?

**Pull request merged**